domains/abubakar.json

```
@@ -0,0 +1,12 @@
+{
+  "description": "My personal portfolio",
+  "repo": "https://github.com/AbubakarKang/AbubakarKang.github.io",
+  "owner": {
+    "username": "AbubakarKang",
+    "email": "",
+    "discord": "TheHacker#4418"
+  },
+  "record": {
+    "CNAME": "https://abubakarkang.github.io/"
```

```suggestion
    "CNAME": "abubakarkang.github.io"
```
cname be a host name plz